nim/happyx/config.yaml

```
@@ -1,6 +1,6 @@
 framework:
   github: HapticX/happyx
-  version: 0.16
+  version: 0.20
```

```suggestion
  version: '0.20'
```
because of yaml parsing in ruby :-)